evalml/automl/automl_search.py

```diff
@@ -831,7 +847,7 @@ def search(self, show_iteration_plot=True):
             % ", ".join([model.value for model in self.allowed_model_families])
         )
         self.search_iteration_plot = None
-        if self.plot:
+        if self.plot and self.verbose:
```

Comment:
If I follow correctly, this means we will only show the search iteration plot in verbose mode.

However, I thought we already had code set up which would only show this plot in the jupyter console. Meaning, if you run evalml outside of jupyter, it will never show up.

Is this correct? If so, why tie this to the verbose flag?

Reply:
I'm happy to change this if people think it's important! I added this limitation since I was testing my changes in jupyter and it felt weird to have just a graph updating at random times, when there was no other output coming during search.

Reply:
Ah right, that makes sense. If we're gonna keep the verbosity flag, I suppose it would be strange to see a plot appear without any log output.